Makefile

```diff
@@ -43,4 +43,4 @@ clean:
 documentation:
 	make project
 	@$(call check_for_gem,jazzy)
-	jazzy --module MongoSwift --module-version 0.0.9 --root-url https://mongodb.github.io/mongo-swift-driver/ --documentation Development.md
+	jazzy --module MongoSwift --module-version 0.0.9 --root-url https://mongodb.github.io/mongo-swift-driver/ --documentation=Guides/*.md
```

Comment:
So _all_ generated documentation goes into `Guides` now?

Reply:
No, the `--documentation` option just isn't named very well. It's used for specifying input files for guides. See the [jazzy docs](https://github.com/realm/jazzy#guides) for more info.

Reply:
d'oh, okay